modules/git-workflow-github.md

```diff
@@ -270,14 +273,17 @@ at (doing a code review), and approve a good solution.
   - Describe the targets in the PR body so that readers can understand
     in more details what the PR is about
   - If your PR resolves an issue just add `Resolves #3` somewhere 
-    in the comment (`3`
-    is a number of an issue you can see beside its title). GitHub will
-    bind your PR to that issue and the issue will get closed
+    in the comment (`3` is a number of an issue you can see beside its title). 
+    GitHub will bind your PR to that issue and the issue will get closed
     (as resolved) once your PR is merged.
 - Draw attention
   - PR created from a fork is bound to the original repo by GitHub,
     so navigate to the original repo, Pull Requests and open a PR
-    you've opened
+    you've opened  
+&nbsp; &nbsp; You may not assign reviewers if you're not an owner of the repo  
+&nbsp; &nbsp; or not a part of organization. Try identifying the owner or repo  
+&nbsp; &nbsp; collaborators and mention them in a comment by their names.  
+&nbsp; &nbsp; If unable to identify relevant persons then just skip this step.  
```

Comment:
`&nbsp;` is obviously excessive. Please replace with a human friendly space

Reply:
@OleksiyRudenko I tried the usual spaces, but it did not work (there was no indentation, probably due to the fact that this is a list item). I did it in the code block for now and I propose like one more option that it could be as a blockquote or is there another way?